src/Components/Footer/Footer.scss

```diff
@@ -1,6 +1,6 @@
-@import '../commonStyles/reset.scss';
-@import '../commonStyles/variables.scss';
-@import '../commonStyles/font-family.scss';
+@import './../../commonStyles/reset.scss';
+@import './../../commonStyles/variables.scss';
+@import './../../commonStyles/font-family.scss';
```

Comment:
there's no need to import reset and font-family to each file. Everything works well even without it